src/resources/postcss/aggregator-page.pcss

```diff
@@ -252,7 +252,57 @@
 	}
 
 	.tribe-notice-tribe-missing-aggregator-license {
-		background: #f7fcfe;
+		background: url(../images/aggregator/ea-upsell-bkg.svg) transparent no-repeat;
+		background-size: cover;
+		border: none;
+		padding: 60px 0;
+		text-align: center;
+
+		.upsell-banner {
+			margin-bottom: 30px;
+			margin-left: auto;
+			margin-right: auto;
+		}
+
+		h3 {
+			color: white;
```

Comment:
[8am musings to follow] ...I would love to have a discussion about this. I just recently started using `white` in lieu of `$white` (which currently exists as a variable).  There is no need to declare the variable as white is white and black is black.  

Does using #fff (my normal choice in a non-sass environment) give performance gains?

...and finally do we have a CSS style guide? I've only gotten as far as PHP in the Engineering Standards. (just looked and noted...can we elaborate on what we have? I love _most_ of what Github is doing with theirs http://primercss.io/)